server/cluster/cluster.go

```diff
@@ -1485,7 +1485,9 @@ func (c *RaftCluster) checkStores() {
 		offlineStore := store.GetMeta()
 		id := offlineStore.GetId()
 		regionSize := c.core.GetStoreRegionSize(id)
-		c.updateProgress(id, store.GetAddress(), removingAction, float64(regionSize), float64(regionSize), false /* dec */)
+		if c.IsPrepared() {
```

Comment:
Is it better to move this check inside the `updateProgress` function?

Reply:
We can avoid calculating the threshold in the preparing stage.